libraries/Netdump/astyle_goodies.conf

```diff
@@ -0,0 +1,32 @@
+# Code formatting rules for Arduino examples, taken from:
```

Comment:
Please remove this file, and add the update you made in `tests/astyle_core.conf`

Reply:
The astyle_goodies.conf is  leftover copying from your repo, will be removed.
Now busy with the layout/astyle.

The tests/restyle.sh doesn't work on my (windows) env.
I will take the astyle_core and astyle.example conf and use them locally for formatting.

Think I am fairly complete on functionality. If you see/need some additional options please let me know.

Will also add Readme.rst

Reply:
Can we help you with running astyle ?
Are you using WSL under windows ?